src/CSI.IBTA.Shared/Utils/PasswordHasher.cs

```diff
@@ -1,19 +1,18 @@
-﻿using CSI.IBTA.AuthService.Interfaces;
-using System.Security.Cryptography;
+﻿using System.Security.Cryptography;
 
-namespace CSI.IBTA.AuthService.Authentication
+namespace CSI.IBTA.Shared.Utils
 {
-    internal class PasswordHasher : IPasswordHasher
+    public static class PasswordHasher
```

Comment:
Lets keep it as is, but for the future  - lest avoid static classes, until they are extensions or super simple. 
They could not be mocked for the unit test which is disadvantage.